src/reports/management/commands/report.py

```diff
@@ -0,0 +1,54 @@
+from django.core.management.base import BaseCommand
+import logging
+from publisher import models, fragment_logic, utils
+
+LOG = logging.getLogger(__name__)
+
+def avl2csv():
+
+    rs = models.ArticleVersion.objects \
+        .select_related('article') \
+        .defer('article_json_v1', 'article_json_v1_snippet') \
+        .order_by('article__manuscript_id', 'version') \
+        .all()
```

Comment:
I think this may find stubs like articles coming from EJP but not yet ingested, or articles that have been related to an existing one but neither yet ingested. This is not necessarily a problem as long as we are aware of these cases, which won't be backfilled.